src/shared_modules/utils/pipelineNodesImp.h

```diff
@@ -31,7 +31,7 @@ namespace Utils
             {}
             ReadNode(Functor functor,
                      const unsigned int numberOfThreads)
-                : DispatcherType{ functor, numberOfThreads }
+                : DispatcherType{ functor, numberOfThreads, 0 }
```

Comment:
Hi @Dwordcito 

Maybe the `UNLIMITED_QUEUE_SIZE` macro could be moved to a common file and used here as well.